frontend/src/components/UserSearch.vue

```diff
@@ -216,68 +245,105 @@ export default {
       this.$router.push({ name: "UserCreate" });
     },
     searchUser: function(queryParameters) {
```

Comment:
I noticed that errors and warnings don't go away on subsequent searches:

![image](https://user-images.githubusercontent.com/1767127/111201729-50a17a00-8580-11eb-9861-88a22afe8cfa.png)

This isn't a bug caused by any of your code, but certainly I'm noticing it now that we have the "max search results" warning. I think errors are cleared by calling this method at the start of the function, so please confirm and add:

```javascript
this.$store.commit("alert/dismissAlert");
```

Reply:
Fixed by https://github.com/bcgov/moh-keycloak-user-management/pull/61/commits/f80cefb45f4fc09a273ef5d12519b050f21e1e52